### PR TITLE
Moved setup to setup.py

### PR DESCRIPTION
**Pull request by andersbogsnes**
Changed to using pip-tools based workflow

### CODE REVIEW

File: dev-requirements.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-rrequirements.txt
```
Comment (thomasfrederikhoeck) [inline dev-requirements.in]: Space here?

File: setup.py [hunk 2]
```diff
@@ -1,4 +1,97 @@
...
+PACKAGES = find_packages(where="src")
+INSTALL_REQUIRES = [
+    "pandas",
+    "scikit-learn",
+    "matplotlib",
+    "pyyaml",
+    "gitpython",
+    "joblib",
+]
+KEYWORDS = ["ml", "framework", "tooling"]
```
Comment (thomasfrederikhoeck) [inline setup.py]: Maybe 'productionize' ;-)

File: tox.ini [hunk 3]
```diff
@@ -36,3 +36,41 @@ deps =
 commands =
     pip wheel -w {envtmpdir}/build --no-deps .
     twine check {envtmpdir}/build/*
+
+[testenv:black]
+skip_install = True
+deps = black
+commands = black .
```
Comment (thomasfrederikhoeck) [inline tox.ini]: no diff command?

Comment (andersbogsnes) [reply tox.ini]: This one is meant for when you want to run black on your code - in the linting step we use --diff

File: setup.py (hunk 2 above)
Comment (andersbogsnes) [reply setup.py]: sure :smile:

File: dev-requirements.in (hunk 1 above)
Comment (andersbogsnes) [reply dev-requirements.in]: generally not needed, but we can add it

**Pull request merged**